default.nix

```diff
@@ -11,6 +11,13 @@ let
   });
   socket-io-src = pkgs.fetchgit (removeAttrs (lib.importJSON ./pkgs/engine-io.json) ["date"]);
   cardano-sl-src = pkgs.fetchgit (removeAttrs (lib.importJSON ./pkgs/cardano-sl.json) ["date"]);
+  cores   = 4;
```

Comment:
These attributes should evaluate to a derivation. Instead threads and cores should be part of the function inputs for default.nix